checkstyle.xml

```diff
@@ -100,7 +100,7 @@
     </module>
 
     <!-- Checks for Naming Conventions. -->
-    <!-- See http://checkstyle.sf.net/config_naming.html -->
+    <!-- See https://checkstyle.org/config_naming.html -->
```

Comment:
https://checkstyle.org/config_naming.html#ConstantName ?

Reply:
No, this comment is also for LocalFinalVariableName, LocalVariableName...